Venmo/src/main/java/com/braintreepayments/api/venmo/VenmoClient.kt

```diff
@@ -335,7 +335,7 @@ class VenmoClient internal constructor(
 
     private val analyticsParams: AnalyticsEventParams
         get() {
-            val eventParameters = AnalyticsEventParams()
+            val eventParameters = AnalyticsEventParams(appSwitchUrl = merchantRepository.returnUrlScheme)
```

Comment:
This may be my understanding of venmo, but would this be the full URL we are attempting to open or just the scheme? I think we want the full URL, but may be misunderstanding this property on android.

Reply:
Ah gotcha, I think what we actually want is this URL instead: https://github.com/braintree/braintree_android/blob/9966290f91eea97bbc27850055c98d91f7368126/Venmo/src/main/java/com/braintreepayments/api/venmo/VenmoClient.kt#L188 It doesn't look like we ever set the return URL as the base URL.

Reply:
Oh! I misinterpreted the AC in the story. So we want to log the URL that we send to the Venmo app/CCT. Thanks for calling this out - I'll make the change.

Reply:
Updated in this commit: 1135c26b105250621b94fd5ce19ef6354abb427f